moe/views/utils.py

```diff
@@ -57,6 +59,31 @@ def _make_covariance_of_process_from_params(params):
     covariance_of_process = covariance_class(hyperparameters)
     return covariance_of_process
 
+def _make_python_covariance_of_process_from_params(params):
```

Comment:
seems like this could be a part of the ``_make_covariance_of_process_from_params`` function. Just have a keyword arg that defaults to cpp which selects btwn accessing ``cpp_covariance_class`` and ``python_covariance_class``

sidenote: the real solution here would be to have the C++ covariance objects exposed fully to Python (which we want to do anyway so that users can select a cov fcn; right now SquareExponential is hard-coded :(). Then you could use them wherever. But I'm not asking you to do that here, that's a big change.